lib/tapjoy/ldap/key/show.rb

```diff
@@ -0,0 +1,25 @@
+module Tapjoy
+  module LDAP
+    module Key
+      # Show all of a user's keys
+      class Show
+        def show
+          username = opts[:user]
```

Comment:
Yeah. I thought it might be a strange user experience if you ask to show a users keys and we respond with absolutely nothing.